plugins/app-backend/src/lib/config.ts

```diff
@@ -87,7 +89,10 @@ export async function readConfigs(options: ReadOptions): Promise<AppConfig[]> {
     const serializedSchema = await fs.readJson(schemaPath);
 
     try {
-      const schema = await loadConfigSchema({ serialized: serializedSchema });
+      const schema = await loadConfigSchema({
+        serialized: serializedSchema,
+        additionalSchemas: options.additionalSchemas,
+      });
```

Comment:
Wanna note here that I think we'll move away from the need for frontend config schema for plugins. The new frontend system basically has all config under `app.*` and I think we may even be able to get rid of the `ConfigApi` completely in the frontend. Basically we hopefully won't need to bother with this in the future.

Reply:
:+1:

Reply:
Had another thing about what we could do here. I'm thinking the best design here might be to introduce an extension point for the app backend, along with an option for the router. I think the best case again would be if we could make that something that provides the whole `ConfigSchema` rather than additional entries. Seems like that shouldn't be too bad? bit of duplicated code for loading the serialized schema perhaps but I think that prolly worth it.